packages/eslint-config-react-app/index.js

```diff
@@ -155,6 +155,15 @@ module.exports = {
     'unicode-bom': ['warn', 'never'],
     'use-isnan': 'warn',
     'valid-typeof': 'warn',
+    'no-restricted-properties': ['warn', {
+      object: 'require',
+      property: 'ensure',
+      message: 'Please use import() instead.'
```

Comment:
@Timer yeah. I would love to. 😄 I can work on the doc. If you like we can add the link to [webpack import() documentation](https://webpack.js.org/guides/code-splitting-import/) for the time being